src/test/resources/config/scriptengines/perl.properties

```diff
@@ -0,0 +1,4 @@
+# General configuration
```

Comment:
You need to add this file with real value to the config/scriptengines directory in the Scheduler